bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/TypeChecker.java

```diff
@@ -203,7 +203,7 @@ public static BType getType(Object value) {
             return BTypes.typeNull;
         } else if (value instanceof Long) {
             return BTypes.typeInt;
-        } else if (value instanceof Double) {
+        } else if (value instanceof Double || value instanceof Float) {
```

Comment:
We haven't used Float anywhere right?

Reply:
yeah.. let me check again. some grpc tests were not working,, gave me a classcsat exception.